### PR TITLE
Change execution stats intervals

### PR DESCRIPTION
**Pull request by scorbajio**
This change switches `vmexecution.ts` stat reporting over to using time instead of block intervals.

### CODE REVIEW

**Comment by scorbajio** [comment]
I tested this by syncing the client executing blocks starting from block `68100` on `mainnet`. The stats seems to be getting reported in 40 seconds intervals and the stats are updating and progressing as expected.

**Comment by holgerd77** [comment]
Updated this via UI

File: packages/client/src/execution/vmexecution.ts [hunk 1]
```diff
@@ -676,10 +687,11 @@ export class VMExecution extends Execution {
       })
 
       if (txHashes.length === 0) {
+        this._statsVm = vm
+
         // we are skipping header validation because the block has been picked from the
         // blockchain and header should have already been validated while putBlock
         const beforeTS = Date.now()
-        this.stats(vm)
```
Comment (scorbajio) [inline packages/client/src/execution/vmexecution.ts:682]: One thing to note about this change is that now stat collection and logging isn't being included in the time measurement of `runBlock`, so there will potentially be a difference in the `time` value being logged for block execution.

**Comment by holgerd77** [comment]
Updated this via UI

**Review by holgerd77 — APPROVED** [review]
Thanks a lot, this looks great! 🙂 

grafik

I think we can (and should) further reduce the frequency, the above is a run on mainnet e.g. and stats still occuring pretty often and still distract from the main progress output.

Have set down from 40 to 90 secs and pushed, will otherwise merge.

**Comment by holgerd77** [comment]
Updated this via UI

**Pull request merged**